File: Backend/Hunter/Models/Game.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Hunter.Models
{
    public class Game
    {
        public int Id { get; set; }
        public int CreatorId { get; set; }
        public DateTime StartDatetime { get; set; }
        public DateTime? EndDatetime { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public string Photo { get; set; }
        public bool Ended { get; set; }
        public int? WinId { get; set; }
        public DateTime? WinTimestamp { get; set; }
        public string WinCode { get; set; }
    }
}

File: Mobile/Android/app/src/main/java/com/utn/frba/desarrollomobile/hunter/service/models/Game.kt
package com.utn.frba.desarrollomobile.hunter.service.models

import com.google.gson.annotations.SerializedName

class Game {
    @SerializedName("id")
    var id: Int = 0
    @SerializedName("creatorId")
    var creatorId: Int = 0
    @SerializedName("startDatetime")
    var startDatetime: String? = null
    @SerializedName("endDatetime")
    var endDatetime: String? = null
    @SerializedName("latitude")
    var latitude: Float = 0.toFloat()
    @SerializedName("longitude")
    var longitude: Float = 0.toFloat()
    @SerializedName("photo")
    var photo: String? = null
    @SerializedName("ended")
    var ended: Boolean? = false
    @SerializedName("winId")
    var winId: Int? = 0
    @SerializedName("winTimestamp")
    var winTimestamp: String? = null
    @SerializedName("winCode")
    var winCode: String? = null
    @SerializedName("clues")
    var clues: Array<String> = emptyArray()
    @SerializedName("userIds")
    var userIds: Array<Int> = emptyArray()
}